packages/auth-browser/src/lib/auth-browser.ts

```diff
@@ -44,14 +46,23 @@ export const checkAndSignAuthMessage = ({
 
   // -- check and sign auth message based on chain
   if (chainInfo.vmType === VMTYPE.EVM) {
+    // TODO: Uncomment below after the new node changes are deployed
+    // if (!litNodeClient) {
+    //   throwError({
+    //     message: 'litNodeClient not provided for EVM signing',
+    //     errorKind: LIT_ERROR.LIT_NODE_CLIENT_NOT_PROVIDED.kind,
+    //     errorCode: LIT_ERROR.LIT_NODE_CLIENT_NOT_PROVIDED.name,
+    //   });
+    // }
+
```

Comment:
Is there a task for this TODO?

Reply:
Good idea lemme create one on Linear and link it here